tests/acceptance/features/webUIFiles/copy.feature

```diff
@@ -97,3 +97,25 @@ Feature: copy files and folders
     And folder "simple-empty-folder" should be listed on the webUI
     And as "user1" folder "folder with space/simple-empty-folder/simple-empty-folder" should exist
     And as "user1" folder "simple-empty-folder" should exist
+
+  Scenario: cancel copying a file
+    Given user "user1" has logged in using the webUI
+    And the user has browsed to the files page
+    When the user tries to copy folder "data.zip" into folder "simple-folder" using the webUI
+    And the user cancels the attempt to copy file into folder "simple-folder" using the webUI
+    Then file "data.zip" should be listed on the webUI
+
```

Comment:
```
And the file "...." should not be listed inside "simple-folder"
```

IMO only checking for data.zip on the files page will not verify if the copy has been canceled successfully.